src/main/controllers/TribunalResponseCYANotSystemUserController.ts

```diff
@@ -0,0 +1,54 @@
+import { Response } from 'express';
+
+import { AppRequest } from '../definitions/appRequest';
+import { ErrorPages, PageUrls, TranslationKeys } from '../definitions/constants';
+import { FormContent } from '../definitions/form';
+import { AnyRecord } from '../definitions/util-types';
+
+import { createDownloadLink } from './helpers/DocumentHelpers';
+import { getPageContent } from './helpers/FormHelpers';
+import { getCancelLink } from './helpers/LinkHelpers';
+import { getRespondentCyaContent } from './helpers/RespondentApplicationCYAHelper';
+import { getLanguageParam } from './helpers/RouterHelpers';
+
+export default class TribunalResponseCYANotSystemUserController {
+  public get(req: AppRequest, res: Response): void {
+    const userCase = req.session?.userCase;
+
+    const content = getPageContent(req, <FormContent>{}, [
+      TranslationKeys.SIDEBAR_CONTACT_US,
+      TranslationKeys.COMMON,
+      TranslationKeys.TRIBUNAL_RESPONSE_CYA,
+    ]);
+
+    const translations: AnyRecord = {
+      ...req.t(TranslationKeys.TRIBUNAL_RESPONSE_CYA, { returnObjects: true }),
+    };
+
+    const downloadLink = createDownloadLink(userCase?.supportingMaterialFile);
+
+    let storedUrl;
+    let id;
+    if (userCase.selectedRequestOrOrder) {
+      storedUrl = ErrorPages.NOT_FOUND; // InterceptPaths.TRIBUNAL_RESPONSE_SUBMIT_CYA + getLanguageParam(req.url);
```

Comment:
wonder if these comments are needed

Reply:
This is a temporary holding page and will be updated in the coming PR